R/microserver.r

```diff
@@ -44,6 +51,10 @@ run_server <- function(routes, port = 8103) {
 
   repeat {
     httpuv::service(1)
+    if (!is.null(.microserver_env[['__decorators']])) {
+      lapply(.microserver_env[['__decorators']], function(decorator) decorator(.microserver_env))
+    }
+    clean_envir(.microserver_env, exceptions = c('__decorators'))
```

Comment:
Actually kind of unsure here. Ideally you'd call the decorators after each request has been served. But looks like httpuv::service is going to serve all requests in the queue?